index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/styles/style.css">
+    <title>capstone</title>
+</head>
+<body>
+    <header>
+        <nav class="top-nav">
+            <ul class="nav-item">
+            <li><a href="#"><img class="nav-img" src="/assets/facebook.png" alt="facebook"></a></li>
+            <li><a href="#"><img class="nav-img" src="/assets/twitter-.png" alt="twitter"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul></nav>
+        <nav class="nav-bar">
+            <a class="d-logo" href="index.html"><span class="logo-span">Y-</span>DEV</a>
+            <a href="#"><img class="humberger" src="/assets/menu.svg" alt="humberger"></a>
+            <ul class="d-nav">
+                <li><a href="pages/about.html">About</a></li>
+                <li><a href="#program">Program</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsor">Sponsor</a></li>
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button class="d-btn">cc campaign</button>
+        </nav>
+        <ul class="mobile-nav">
+               <li><a class="nav-links" href="pages/about.html">About</a></li>
+                <li><a class="nav-links" href="#program">Program</a></li>
+                <li><a class="nav-links" href="#join">Join</a></li>
+                <li><a class="nav-links" href="#sponsor">Sponsor</a></li>
+                <li><a class="nav-links" href="#news">News</a></li>
+        </ul>
+        <div class="times">&times</div>
+    </header>
+    <section id="join" class="hero">
+        <p class="p-1">"Hello! Dev World"</p>
+        <h1 class="h-1">YOUNG DEV'S GLOBAL SUMMIT UGANDA</h1>
+        <p class="p-2">A joyful celebration of all young youth in the development industry through spreading love and sharing ideas about tech.</p>
+        <h2 class="h-2">2023.23.04(WED) ~ 24(THURS)</h2>
+        <P class="p-3">@ Global Dev World, Kampala,Uganda</P>
+    </section>
```

Comment:
- [ ] Kindly make your font weight bolder, also the border of your first paragraph here, is colored white in the original design, and the main title is shorter in width. Check the images below:
your design:
![image](https://user-images.githubusercontent.com/91486791/234955881-6b2b3e84-a45e-45d2-b966-832a1ed0b27f.png)
actual:
![image](https://user-images.githubusercontent.com/91486791/234955988-620eea6b-eeaa-4840-8ff0-239b34552444.png)
- [ ]  Please give more padding before this section:
![image](https://user-images.githubusercontent.com/91486791/234958900-8853325b-3e65-4dc7-a36c-4d1ae5cd6cd7.png)
![image](https://user-images.githubusercontent.com/91486791/234958970-71328198-251f-4ff4-b662-d5d11c2330aa.png)